scripts/run.sh

```diff
@@ -30,20 +27,30 @@ source "$SUBNET_EVM_PATH"/scripts/versions.sh
 source "$SUBNET_EVM_PATH"/scripts/constants.sh
 
 VERSION=$avalanche_version
+
+# "ewoq" key, do not include "0x"
 DEFAULT_ACCOUNT="0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC"
 GENESIS_ADDRESS=${GENESIS_ADDRESS-$DEFAULT_ACCOUNT}
 
 SKIP_NETWORK_RUNNER_START=${SKIP_NETWORK_RUNNER_START:-false}
-SKIP_NETWORK_RUNNER_SHUTDOWN=${SKIP_NETWORK_RUNNER_SHUTDOWN:-false}
+SKIP_NETWORK_RUNNER_SHUTDOWN=${SKIP_NETWORK_RUNNER_SHUTDOWN:-true}
 RUN_SIMULATOR=${RUN_SIMULATOR:-false}
 ENABLE_SOLIDITY_TESTS=${ENABLE_SOLIDITY_TESTS:-false}
-AVALANCHE_LOG_LEVEL=${AVALANCHE_LOG_LEVEL:-INFO}
+AVALANCHE_LOG_LEVEL=${AVALANCHE_LOG_LEVEL:-WARN}
 ANR_VERSION=$network_runner_version
 GINKGO_VERSION=$ginkgo_version
-GINKGO_FOCUS_FLAGS=${GINKGO_FOCUS_FLAGS}
-GINKGO_SKIP_FLAGS="\[Precompiles\]"
+
+# by default, "run.sh" should not run any tests...
+# simulator tests are not implemented in ginkgo
+# it instead runs external binary "simulator"
+# so we exclude all e2e tests here
+# ref. https://onsi.github.io/ginkgo/#spec-labels
+GINKGO_LABEL_FILTER="!precompile-upgrade && !solidity-with-npx && !solidity-counter"
+if [[ ${RUN_SIMULATOR} == true ]]; then
+  GINKGO_LABEL_FILTER="ping"
+fi
```

Comment:
Why do we set the label filter to `ping` if we are running the simulator? Is this intended to send a ping and then let the network sit and continue to run?

Reply:
`ping` tests basically no test other than pinging the network runner server. We are trying to skip all actual tests and just run simulator. Will document.